subprojects/docs/src/docs/userguide/running-builds/configuration_cache.adoc

```diff
@@ -1048,3 +1048,40 @@ Alternatively, for Kotlin implementations, the Kotlin compiler can be configured
 include::sample[dir="snippets/configurationCache/samConversion/groovy",files="build.gradle[tags=configure]"]
 include::sample[dir="snippets/configurationCache/samConversion/kotlin",files="build.gradle.kts[tags=configure]"]
 ====
+
+[[config_cache:not_yet_implemented:accessing_top_level_at_execution]]
+=== Accessing top-level methods and variables of a build script at execution time
+
+Currently, Gradle cannot run a task restored from the configuration cache if a task action calls a method defined directly in the build script.
```

Comment:
```suggestion
Gradle cannot run a task restored from the configuration cache if a task action calls a method defined directly in the build script.
```
Suggestion: remove "currently" for same reason as "yet" above. "Currently" is implicit in the "current" version of the docs, after all.

Consider rephrasing this to a less technical voice since it is the introductory sentence of the section. Perhaps:

"Methods and variables defined at the top level of your build script are not compatible with the configuration cache. Any tasks that use top-level methods or variables cannot store outputs in the cache."

... or something similar, if I've missed any technical details.